soc/xtensa/intel_apl_adsp/bootloader.cmake

```diff
@@ -2,29 +2,27 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-if(EXISTS ${BOARD_DIR}/bootloader/CMakeLists.txt)
+if(EXISTS ${SOC_DIR}/${ARCH}/${SOC_FAMILY}/common/bootloader/CMakeLists.txt)
```

Comment:
doesn't this path always exist, in what cases we do not have this file?

Reply:
I made it more simple.